test/utils.js

```diff
@@ -64,6 +67,7 @@ function indexToPosRaw(str, index) {
     switch (char) {
       case '\n':
         if (prevChar === '\r') break;
+      // eslint-disable-next-line no-fallthrough
```

Comment:
A comment should suffice (see [here](https://eslint.org/docs/rules/no-fallthrough)):
```suggestion
      // fall through
```